static/sass/styles.scss

```diff
@@ -48,11 +48,19 @@
   max-width: none;
 }
 
-.empty-state-message {
-  .icon {
-    height: 100%;
-    width: 35%;
-  }
+.u-empty-state-icon {
```

Comment:
I just realized now, the `u-` prefix is meant for utility classes, not sure if it's needed here.
I started to group the CSS in files and import them, maybe add one for the empty state?
Then in the file, wrap the whole content with a class and give that to the outer wrapper of the `EmptyState` component. This way the content of the file can't leak into other use cases and the browser can ignore (in all other cases) all the content in that file easily by just looking at the wrapper class.